src/main/resources/assets/additionalguns/models/item/val.json

```diff
@@ -1,457 +1,619 @@
 {
-	"credit": "Made with Blockbench by ThatAvocado_Boi",
+	"credit": "Made with Blockbench",
 	"texture_size": [32, 32],
 	"textures": {
-		"3": "additionalguns:item/val"
+		"1": "additionalguns:item/anvil",
```

Comment:
Why not specify the path to the Minecraft texture instead? Redistributing Minecraft textures is against the EULA anyway
```suggestion
		"1": "minecraft:item/anvil",
```